pkg/manifests/manifests.go

```diff
@@ -1065,7 +1069,11 @@ func (f *Factory) ThanosRulerAlertmanagerConfigSecret() (*v1.Secret, error) {
 		return nil, err
 	}
 
-	s.StringData["alertmanagers.yaml"] += "\n" + string(additionalConfig)
+	if alertmanagerEnabled {
+		s.StringData["alertmanagers.yaml"] += "\n" + string(additionalConfig)
+	} else {
+		s.StringData["alertmanagers.yaml"] = `alertmanagers:` + "\n" + string(additionalConfig)
```

Comment:
just curious, why we care additional config for alter manager while it is disabled? 🤔